iconrpcserver/dispatcher/default/node.py

```diff
@@ -69,16 +68,6 @@ class NodeDispatcher:
         channel_infos = await StubCollection().peer_stub.async_task().get_channel_infos()
         return {"channel_infos": channel_infos}
 
-    @staticmethod
-    @methods.add
-    async def node_announceConfirmedBlock(**kwargs):
```

Comment:
Does not need validate of `announceConfirmedBlock` too?

Reply:
Thanks for review!